Comparison Engine/Comparison Engine/Base Classes/BarManager.cs

```diff
@@ -50,5 +50,21 @@ public float FindDrinkPrice(int barID, int drinkID)
         {
             return barList[barID].availableDrinks;
         }
+
+        //Returns specific Bar obejcts by some property
+        public Bar GetBarByID(int barID)
+        {
+            return barList[barID];
```

Comment:
I might have missed something, but I think we talked about how we can't calls objects by index in the list since if we remove something, the indexing will break